flytekit/exceptions/user.py

```diff
@@ -8,6 +8,22 @@ class FlyteUserException(_FlyteException):
     _ERROR_CODE = "USER:Unknown"
 
 
+class FlyteUserRuntimeException(_FlyteException):
+    _ERROR_CODE = "USER:RuntimeError"
+
+    def __init__(self, exc_value):
+        """
+        FlyteUserRuntimeException is thrown when a user code raises an exception.
+        @param exc_value: The exception that was raised from user code.
+        """
+        self._exc_value = exc_value
+        super(FlyteUserRuntimeException, self).__init__(str(exc_value))
```

Comment:
In Python 3, the inner part is not required anymore:

```suggestion
        super().__init__(str(exc_value))
```

Reply:
done